platform/string_storage_base.cpp

```diff
@@ -97,6 +97,14 @@ void StringStorageBase::SetValue(string const & key, string && value)
   Save();
 }
 
+void StringStorageBase::Update(std::map<std::string, std::string> const & values)
```

Comment:
Why not using the existing SetValue?

Reply:
It rewrites `settings.ini` on every call. I thought that rewriting `setings.ini` ten times is not a good idea.